### PR TITLE
Task-41306 : Add source column on UserManegement

### PR DESCRIPTION
**Pull request by ahmadmido**
This PR will adds a new column on UserManegmenet table -> Source. This column will check if a user is an internal user or a synchronized user, if is internal we display an icon with eXo logo and if's synchronized LDAP/AD we display an icon with a tag SYNC.

### CODE REVIEW

**Review by rdenarie — DISMISSED** [review]

**Review by rdenarie — DISMISSED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/IDMUsersManagement/Style.less [hunk 1]
```diff
@@ -28,6 +28,21 @@
...
+    font-size: 8px;
+    font-family: Arial, Helvetica, sans-serif;
+  }
+
+  .internalIconClass {
+    color: @yellow;
+    display: flex;
+    padding: 5px;
+    font-size: 11px;
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/social/portlets/IDMUsersManagement/Style.less:43]: do you need font-family ?

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/IDMUsersManagement/Style.less [hunk 2]
```diff
@@ -28,6 +28,21 @@
     color: @errorColor;
   }
 
+  .synchronizedIconClass {
+    display: flex;
+    padding: 5px;
+    font-size: 8px;
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/social/portlets/IDMUsersManagement/Style.less:35]: do you need font-family ?

**Review by azayati — CHANGES_REQUESTED** [review]
See review

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**